test/PEPS.jl

```diff
@@ -47,157 +47,16 @@ x, y = m, n
 
 for origin ∈ (:NW, :SW, :WS, :WN, :NE, :EN, :SE, :ES)
 
-    @info "testing peps" origin
-
     peps = PepsNetwork(x, y, fg, β, origin)
     @test typeof(peps) == PepsNetwork
 
-    @info "contracting MPOs (up -> down)"
-
-    ψ = MPO(PEPSRow(peps, 1))
-
-    for A ∈ ψ @test size(A, 2) == 1 end
-
-    for i ∈ 2:peps.i_max
-        println("row -> ", i)
-
-        R = PEPSRow(peps, i)
-        W = MPO(R)
-        M = MPO(peps, i-1, i)
+    ψ = idMPS(peps.j_max)
```

Comment:
If `idMPS` is needed for testing, please do not implement it in the package. We should not put methods needed only in tests in the main code.

Reply:
This is not only for testing, we need the identity MPS for sure.

Reply:
Ok, I'll introduce an `IdentityMPS` and `IdentityMPO` types after this is merged.